app/src/org/commcare/android/database/global/GlobalDatabaseUpgrader.java

```diff
@@ -50,6 +54,15 @@ private boolean upgradeOneTwo(SQLiteDatabase db, int oldVersion, int newVersion)
     }
 
     private boolean upgradeTwoThree(SQLiteDatabase db) {
```

Comment:
Ok cool then I'll split it out into 2 upgrades, there's no dependency that I can think of.